tests/sparseml/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -76,8 +78,15 @@ def test_sparse_model_reload(compressed, config, dtype, tmp_path):
         tmp_path / "oneshot_out", torch_dtype=dtype
     )
 
-    inferred_global_sparsity = SparsityConfigMetadata.infer_global_sparsity(model)
-    assert math.isclose(inferred_global_sparsity, 19.6562, rel_tol=1e-3)
+    def compute_tensor_sparsity(t: torch.Tensor):
+        return t.eq(0).sum().item() / t.numel()
```

Comment:
We have a function for this, it's called `tensor_sparsity` here https://github.com/neuralmagic/sparseml/blob/c234638a1d7bd276c18bdc46005434fca88f9881/src/sparseml/pytorch/utils/helpers.py#L556